mapadroid/worker/WorkerQuests.py

```diff
@@ -675,27 +652,24 @@ def _current_position_has_spinnable_stop(self, timestamp: float):
                 return fort_type == 1 and enabled and not closed and cooldown == 0, False
         # by now we should've found the stop in the GMO
         # TODO: consider counter in DB for stop and delete if N reached, reset when updating with GMO
-        if stop_found:
-            self.logger.warning("Unable to confirm the stop being spinnable - likely not standing exactly on top ...")
-            self._spinnable_data_failcount = 0
-        else:
-            self.logger.warning("Unable to find the stop closeby!")
-            self._spinnable_data_failure()
+        self.logger.warning("Unable to confirm the current location yielding a spinnable stop - likely not standing "
+                            "exactly on top ...")
+        self._spinnable_data_failcount = 0
```

Comment:
I probably need to review how failcount is used but it seems weird to see an issue and set failcount to 0. This is existing code so its probably fine